Remotion/Data/DomainObjects.Security.UnitTests/SecurityClientTransactionExtensionTests/TestHelper.cs

```diff
@@ -150,7 +150,7 @@ public void ExpectObjectSecurityStrategyHasAccess (SecurableObject securableObje
               Arg.Is (_stubUser),
               Arg<IReadOnlyList<AccessType>>.List.Equal (new[] { AccessType.Get (accessTypeEnum) })))
           .WhenCalled (mi => CheckTransaction())
-          .Do (doDelegate);
+          .Do (doDelegate).Repeat.Once();
     }

```

Comment:
Why do we need all these "Repeat.Once()" calls?

Reply:
This change is needed when upgrading from Rhino.Mocks 3.6.0 to 3.6.1.

Reply:
As discussed.